city_metrix/layers/layer.py

```diff
@@ -54,31 +55,64 @@ def groupby(self, zones, layer=None):
         """
         return LayerGroupBy(self.aggregate, zones, layer, self.masks)
 
-    def write(self, bbox, output_path, tile_degrees=None, **kwargs):
+    def write(self, bbox, output_path, tile_degrees=None, buffer_meters=None, **kwargs):
```

Comment:
What do you think about setting the tile size in meters as well? Seem more intuitive for most people and would be easier to compare to the bugger size.

Reply:
I strongly agree, but didn't want to take that on for this ticket since there may be some complexity and it's out of scope.

Could you please create a new ticket and send to me?

Reply:
https://gfw.atlassian.net/browse/CIF-293?atlOrigin=eyJpIjoiOTQ4MGJmNDE5OGE3NGYzYjllNmEwNDY0MTE3YzllMTQiLCJwIjoiaiJ9